notebooks/region_search/Region Searching Workbook.ipynb

```diff
@@ -2794,6 +2813,22 @@
     "del tmpdf"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 74,
+   "id": "bfb23aef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def ra_dec_search_overlapping_sets(df, overlapping_sets):\n",
+    "    \"\"\"\n",
+    "    2/6/2024 COC\n",
+    "    Implementing an extremely basic (RA, Dec) query functionality.\n",
+    "    This will work within the overlapping_sets framework.\n",
+    "    \"\"\"\n",
+    "    set_to_corners"
```

Comment:
It looks like this function is broken? If you want to leave it in as an interface, you could replace set_to_corners with pass for now